test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,89 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static Object nullObject = null;
+    private static Tag tag1;
+    private static Tag tag1Copy;
+    private static Tag tag2;
+
+    @Before
+    public void setup() {
+        tag1 = buildTag("tag1");
+        tag1Copy = buildTag("tag1");
+        tag2 = buildTag("tag2");
+    }
+
+    private static Tag buildTag(String tagName) {
+        try {
+            return new Tag(tagName);
+        } catch (IllegalValueException e) {
+            throw new RuntimeException("test tag name should be valid", e);
```

Comment:
or just add a `throws Exception` to the method signature. If the exception happens, it will result in a test failure, which is what we want.